AdamSensors/v1/homeoffice_pandemic_test.yaml

```diff
@@ -0,0 +1,31 @@
+# The AdamSensors CSC reads voltages off of the ADAM 6024's six analog input channels, in
```

Comment:
Please, rename this file to something like `initial_test_config.yaml`, as suggested above?